README.md

```diff
@@ -1,5 +1,48 @@
 ![Build project](https://github.com/netty/netty/workflows/Build%20project/badge.svg)
 
+# Siren fork of Netty
+
+## Changes
+
+All the parameters `io.netty.*` were prefixed with the `siren` keyword in order to avoid conflict with users possibly setting Netty's original parameters.
```

Comment:
What happens if the user sets a `io.netty.*` parameter ? Are the ignored completely ? Or only if there is a `siren.io.netty.*` parameter set ?